types/xrm/index.d.ts

```diff
@@ -5303,8 +5303,12 @@ declare namespace Xrm {
             /**
              * ID of the selected stage in business process instance.
              */
-            selectedStageId?: string | undefined;
-        }
+             selectedStageId?: string | undefined;
+            /**
+             * Sets the focus on the tab of the form.
+             */
+             tabName?: string | undefined;
+            }
```

Comment:
Can you make sure the closing brackets align? Thank you.

Reply:
Of course, my bad